openrtb_ext/bid.go

```diff
@@ -74,8 +74,8 @@ type ExtBidPrebidMeta struct {
 
 // ExtBidPrebidVideo defines the contract for bidresponse.seatbid.bid[i].ext.prebid.video
 type ExtBidPrebidVideo struct {
-	Duration        int    `json:"duration"`
-	PrimaryCategory string `json:"primary_category"`
+	Duration        int    `json:"duration,omitempty"`
+	PrimaryCategory string `json:"primary_category,omitempty"`
```

Comment:
Good catch! 
I think we need to fix unit tests instead. Here is why:
1) We will not hide known issues in unit tests.
2) We will not modify core structures to satisfy unit tests.
3) We will have unit tests in this PR that will show how to use this new feature and set an examples for future contributors.

I only had to fix 3 files to make unit tests pass:
/adapters/freewheelssp/freewheelssptest/exemplary/multi-imp.json
/adapters/yeahmobi/yeahmobitest/exemplary/simple-video.json
/adapters/pubmatic/pubmatictest/exemplary/video.json

Change in adapter json tests is expected here. I'll confirm this with the team just in case :)

Reply:
Thank you, pushed another commit efd1719 to fix adapter tests and remove the change in bid.go